example/python_examples/munki_manifest.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from helloit_lib import HelloIT
+import plistlib
+
+h = HelloIT()
+
+
+def munki_manifest():
+    try:
+        with open("/Library/Managed Preferences/ManagedInstalls.plist", "rb") as fp:
+            pl = plistlib.load(fp)
+            return pl["ClientIdentifier"]
```

Comment:
might want to do a `.get` here instead since not everyone sets this value.

Reply:
I never got a notification on this, sorry for the late reply! I switched the method around which should make this more universal.

Reply:
I appreciate the review. One thing I still cant get to work right is implementing onClickAction. The example I have copies the value to the clipboard, which is great, except it does this everytime the script runs which is obviously not ideal.

Reply:
I'm pretty sure I have that working in our python3 helloit code. When I'm back from paternity leave I'll look some more. 

Message me on macadmins slack if you can so I don't forget!